docs/pages/guides/customizing-metro.md

```diff
@@ -51,7 +51,7 @@ By default, Metro uses [`uglify-es`](https://github.com/mishoo/UglifyJS) to mini
 **metro.config.js**
 
 ```js
-const { getDefaultConfig } = require('expo/metro-config');
```

Comment:
iirc we were keeping it like `@expo/metro-config` since this doc is unversioned and `expo/metro-config` was added in v41. 
Can merge after dropping SDK 40 docs.